src/scripts/discord.js

```diff
@@ -1,31 +1,55 @@
 const discordrpc = require("discord-rpc");
-const { ipcMain } = require("electron");
-const electron = require("electron");
+const { app, ipcMain } = require("electron");
 const globalEvents = require("../constants/globalEvents");
 const clientId = "833617820704440341";
 const mediaInfoModule = require("./mediaInfo");
 const discordModule = [];
 
+function timeToSeconds(timeArray) {
+  let minutes = (timeArray[0] * 1);
+  let seconds = (minutes * 60) + (timeArray[1] * 1);
+  return seconds;
+}
+
 let rpc;
 const observer = (event, arg) => {
   if (mediaInfoModule.mediaInfo.status == "paused" && rpc) {
     rpc.setActivity(idleStatus);
   } else if (rpc) {
-    rpc.setActivity({
-      ...idleStatus,
-      ...{
-        details: `Listening to ${mediaInfoModule.mediaInfo.title}`,
-        state: mediaInfoModule.mediaInfo.artist,
-        buttons: [{ label: "Play on Tidal", url: mediaInfoModule.mediaInfo.url }],
-      },
-    });
+    let csec = timeToSeconds(mediaInfoModule.mediaInfo.current.split(":"));
```

Comment:
these can be consts.
Would also prefer full names instead of abbreviations.

Reply:
Understandable I will think about a better name